product_docs/docs/pgd/5.6/nodes/subscriber_only/optimizing-so.mdx

```diff
@@ -32,42 +32,44 @@ The write leader nodes in other non-subscriber-only groups replicate changes to
 
 ## Group leaders in the optimized topology
 
- With the optimized topology enabled, only the subscriber-only group's group leader receives changes from other data group's write leaders in the cluster and it takes on the responsibility of replicating those changes to the other nodes in the subscriber-only group.
+With the optimized topology enabled, only the subscriber-only group's group leader receives changes from other data groups' write leaders in the cluster. The group leader takes on the responsibility of replicating those changes to the other nodes in the subscriber-only group.
 
 The other voting nodes choose the group leader from a subscriber-only group's nodes.
-Once selected, the whole cluster becomes aware of the change and any data group's write leaders then replicate data only to this newly selected group leader node. 
+Once the group leader is selected, the whole cluster becomes aware of the change, and any data group's write leaders then replicate data only to this newly selected group leader node. 
 Other data nodes in the data groups don't replicate data to the subscriber-only group's nodes.
 
 This approach avoids the explosion of active connections that can happen when there are large numbers of SO nodes and reduces the amount of replication traffic.
 
-The Subscriber-Only node and group form the building block for PGD tree topologies.
+The subscriber-only node and group form the building block for PGD tree topologies.
 
 ## Enabling the optimized model
 
 By default, PGD 5.6 forces the full mesh topology. This means the optimization described here is off.
-To enable the optimized topology, you must have your data-nodes in subgroups, with proxy routing enabled on the sub groups.
+To enable the optimized topology, you must have your data nodes in subgroups, with proxy routing enabled on the subgroups.
 You can then set the GUC [`bdr.force_full_mesh`](/pgd/5.6/reference/pgd-settings#bdrforce_full_mesh) to `off` to allow the optimization to be activated.
-Note: This GUC needs to be set in the postgresql.conf file on each data node and each node restarted for the change to take effect.
 
-If any requirements of the optimized topology aren't met, the nodes in a subscriber-only group automatically revert to the full mesh topology. When this happens, you'll find in the logs of the nodes in the cluster messages that explain why the optimization was not possible, such as:
+!!! Note
+    This GUC needs to be set in the `postgresql.conf` file on each data node and each node restarted for the change to take effect.
+
+If any requirements of the optimized topology aren't met, the nodes in a subscriber-only group revert to the full mesh topology. When this happens, you'll find in the logs of the nodes in the cluster messages why the optimization wasn't possible, such as:
```

Comment:
Please check this edit. There looked to be extra words here. I think this is what was meant.